assemblies/assembly-about-rhdh.adoc

```diff
@@ -12,11 +12,20 @@ endif::[]
 
 :context: about-rhdh
 
-{product} ({product-very-short}) is an enterprise-grade internal developer portal designed to simplify and streamline software development processes. Combined with {ocp-brand-name}, {product-very-short} empowers platform engineering teams to create customized portals that improve developer productivity, accelerate onboarding, and enable faster application delivery. By reducing friction and complexity, {product-very-short} allows developers to focus on writing high-quality code while adhering to enterprise-class best practices.
+{product} ({product-very-short}) is an enterprise-grade internal developer portal (IDP) that simplifies and accelerates software delivery. It provides a customizable web-based interface that centralizes access to key development resources, including source code repositories, CI and CD pipelines, APIs, documentation, and runtime environments. The platform supports cloud-native environments such as Kubernetes, {ocp-brand-name}, and hybrid infrastructure.
 
-{product-very-short} integrates software templates, pre-designed solutions, and dynamic plugins into a centralized platform, providing tailored solutions for operations and development teams in a unified environment.
+The result is faster delivery, reduces tool sprawl, and standardized practices across teams.
+
+Designed for enterprise-scale software teams, {product-very-short} offers a fully supported, turnkey developer portal experience. It features simplified installation, dynamic plugin management, and seamless integration with existing tools. With enterprise grade security, role-based access control, and 24x7 support, teams can focus on building software instead of maintaining internal infrastructure.
```

Comment:
```suggestion
Designed for enterprise-scale software teams, {product-very-short} offers a fully supported, turnkey developer portal experience. Key features include simplified installation, dynamic plugin management, and seamless integration with existing tools. With enterprise-grade security, role-based access control, and 24x7 support, teams can focus on building and delivering software rather than maintaining internal infrastructure.


```